### DISCUSSION TITLE
Extra space on project names on the trending page

### DISCUSSION THREAD

**Discussion by corentinbettiol** [community/community#86640] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I was browsing the [trending page](https://github.com/trending) (thanks for this), and I noticed an extra space on the links just before all the projects names.

It bothers me a little, so I thought I'd report it here.

![image](https://github.com/community/community/assets/45763865/ac4ed631-2774-46ec-978f-90c63a67dff8)

**Comment by jadfost** [comment]
That´s interesting
